pkg/util/yaml.go

```diff
@@ -18,9 +18,11 @@ import (
 	"bytes"
 	"strings"
 
-	"github.com/gogo/protobuf/jsonpb"
-	"github.com/gogo/protobuf/proto"
+       "github.com/gogo/protobuf/proto"
```

Comment:
I think this is wrong - and the logic to convert one json to another may really just be overhead that is unneeded.